.github/workflows/job_PreCommit_PythonAutoformatter.yml

```diff
@@ -0,0 +1,27 @@
+name: Precommit Python Formatter
+run-name: Python Formatter
+on:
+  pull_request:
+    branches: [ "master", "release-*" ]
+    paths: [ "sdks/python/apache_beam/.*"]
+  issue_comment:
+    types: [created, edited]
+
+
+jobs:
+  PythonFormatter:
+    if: github.event.issue.pull_request && contains(github.event.comment.body, 'Run PythonLint PreCommit')
```

Comment:
Could we add something like https://github.com/apache/beam/blob/master/.test-infra/jenkins/README.md here?

Reply:
Basically catalog all the jobs in Github Actions? Same structure?

Reply:
Yeah, one of the main purpose is to look up the trigger phase of the workflow (if defined)

Reply:
Added one in https://github.com/apache/beam/pull/26968/commits/f3ea2d8a68897e1446f5cb0c25eadb126b39e4c6

Lets discuss what exactly is this for and any updates to design on the meet